tests/threads/test_web_api_handler.py

```diff
@@ -0,0 +1,89 @@
+from typing import Generator
```

Comment:
`typing.Generator`は非推奨のエイリアスです。
```suggestion
from collections.abc import Generator
```
https://docs.python.org/ja/3/library/typing.html#typing.Generator